src/odemis/acq/test/move_test.py

```diff
@@ -710,6 +897,55 @@ def test_lin_rot_axes(self):
         act_dist = _getDistance(point2, point1)
         self.assertAlmostEqual(exp_dist, act_dist)
 
+    def test_get_progress(self):
+        """
+        Test getMovementProgress function behaves as expected
+        """
+        start_point = {'x': 0, 'y': 0, 'z': 0}
+        end_point = {'x': 2, 'y': 2, 'z': 2}
+        current_point = {'x': 1, 'y': 1, 'z': 1}
+        progress = getMovementProgress(current_point, start_point, end_point)
+        self.assertTrue(util.almost_equal(progress, 0.5, rtol=RTOL_PROGRESS))
+
+        current_point = {'x': .998, 'y': .999, 'z': .999}  # slightly off the line
+        progress = getMovementProgress(current_point, start_point, end_point)
+        self.assertTrue(util.almost_equal(progress, 0.5, rtol=RTOL_PROGRESS))
+
+        # Same in the other direction
+        act_rot_error = _getDistance(point1, point2)
+        self.assertAlmostEqual(exp_rot_error, act_rot_error)
```

Comment:
Where is exp_rot_error assigned?

Reply:
Sorry this is not my code, I will align with the master